eng/common/instructions/azsdk-tools/check-package-validation.instructions.md

```diff
@@ -0,0 +1,41 @@
+---
+description: 'Check if the package validation checks have passed for the SDK package.'
+---
+## Goal
+Check the validation checks for the SDK package by collecting the required information from the user and executing the run_check command.
+
+## Instructions
+1. **Collect Required Information**:
+    - Prompt the user for the exact package name
+    - Prompt the user to select the programming language from the following options (case sensitive):
```

Comment:
Would it be more flexible to say something like below? That way we aren't hard-coding the enum values for language and check type which could drift in the future. OR I think we should consider auto-generating these instructions (or leveraging `[Description]` instead).

> Prompt the user specify values for each of the parameters required by the `azsdk_package_run_check` tool. Present the user with options to pick from for the allowed values specified by the parameter schema.